packages/apidom-ls/src/config/openapi/openapi3_1/completion.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  ApidomCompletionItem,
+  CompletionFormat,
+  CompletionType,
+} from '../../../apidom-language-types';
+
+const completion: ApidomCompletionItem[] = [
```

Comment:
just a nitpick here - it's better to define the fields in the same order as they are defined in the spec.